src/main/java/uk/gov/pay/adminusers/resources/InviteResource.java

```diff
@@ -129,6 +129,19 @@ public Invite updateInvite(@Parameter(example = "d02jddeib0lqpsir28fbskg9v0rv")
                 .collect(Collectors.toList());
         return inviteService.updateInvite(inviteCode, updateRequests);
     }
+    
+    @POST
+    @Path("/v1/api/invites/{code}/send-otp")
+    @Produces(APPLICATION_JSON)
+    public void sendOtp(@Parameter(example = "d02jddeib0lqpsir28fbskg9v0rv") @PathParam("code") String inviteCode) {
```

Comment:
we should add some `@ApiResponse` annotations to this so we document the possible responses correctly in the OpenApi file